tests/robot-tests/tests/general_public/methodologies_page.robot

```diff
@@ -30,6 +30,27 @@ Validate page contents
     ...    Pupil absence statistics: methodology
     ...    %{PUBLIC_URL}/methodology/pupil-absence-in-schools-in-england
 
+Navigate to /methodology page (prod)
+    [Tags]    NotAgainstLocal    NotAgainstDev    NotAgainstPreProd
+    user navigates to public methodologies page
+    user waits until page contains
+    ...    Browse to find out about the methodology behind specific education statistics and data and how and why they're collected and published.
+    ...    10
+
+Validate accordion sections exist (prod)
+    [Tags]    NotAgainstLocal    NotAgainstDev    NotAgainstPreProd
+    user waits until page contains accordion section    Children's social care
+    user waits until page contains accordion section    COVID-19
+    user waits until page contains accordion section    Destination of pupils and students
+    user waits until page contains accordion section    Early years
+    user waits until page contains accordion section    Finance and funding
+    user waits until page contains accordion section    Further education
+    user waits until page contains accordion section    Higher education
+    user waits until page contains accordion section    Pupils and schools
+    user waits until page contains accordion section    School and college outcomes and performance
+    user waits until page contains accordion section    Teachers and school workforce
+    user waits until page contains accordion section    UK education and training statistics
+
 Validate Related information section links exist
```

Comment:
As this file stands, this is the only test case that will run against Dev - but I think this test case might still pass because `user opens browser` leaves the user on the home page, and I think these links can be found there too?

You could check you're on the right page with `user checks url contains`.